crates/edr_eth/src/state.rs

```diff
@@ -1,3 +1,5 @@
+pub use revm_primitives::state::*;
```

Comment:
Is this intentional? I'd prefer to use explicit imports if possible

Reply:
I'm trying to find a good way to re-export the `revm_primitives` types to avoid other dependent crates or users from also having to import the exact same version. The simplest way I found is to re-export the public modules, but this public modules happens to have the same names as one of our own modules, `state`. Hence, why I re-exported all of its content in that particular sub-module.

Do you have any alternative proposals?

Reply:
Ok thanks it makes sense to have a wildcard import for this use case.